firedrake/functionspaceimpl.py

```diff
@@ -20,6 +20,45 @@
 from firedrake.petsc import PETSc
 
 
+def check_element(element, top=True):
+    """Run some checks on the provided element.
```

Comment:
We are trying to slowly shift towards using numpydoc and type hints (see the [policy doc](https://github.com/firedrakeproject/firedrake/wiki/Docstrings)). Any chance these docstrings could be done in that format?

Reply:
This is not new code, it was moved from functionspace.py. I think this should be addressed separately.

Reply:
On top of that, docs are not currently building properly, we should wait until they get fixed.